crates/bevy_core_pipeline/src/lib.rs

```diff
@@ -176,7 +176,7 @@ impl Plugin for CorePipelinePlugin {
                 draw_3d_graph::node::MAIN_PASS,
                 MainPass3dNode::IN_VIEW,
             )
-            .unwrap();
+            .expect("Could not add slot edge to `RenderGraph`.");
```

Comment:
Something to consider before throwing expects onto more "internal" non-user facing code: static strings are compiled into the binary. I don't think we want to bloat bevy apps with a bunch of "internal error messages", especially in cases like these where the context of the code is as good (if not better) for debugging than the message in the expect (for a Bevy Engine dev debugging a user-reported error).